doc/website/sidebars.js

```diff
@@ -21,6 +21,15 @@ module.exports = {
                 'getting-started/getting-started-apply',
             ]
         },
+        {
+            type: "category",
+            label: "Declarative Workflows",
+            collapsed: false,
+            items: [
+                {type: 'doc', id: 'declarative/inspect', label: 'schema inspect'},
+                {type: 'doc', id: 'declarative/apply', label: 'schema apply'},
```

Comment:
```suggestion
                {type: 'doc', id: 'declarative/apply', label: 'Applying Changes'},
```

Commands can go in the CLI reference.